lib/ui/academics/OccupationList.dart

```diff
@@ -0,0 +1,229 @@
+import 'dart:math' as math;
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:illinois/model/occupation/Occupation.dart';
+import 'package:illinois/model/occupation/Skill.dart';
+import 'package:illinois/service/skills/OccupationsService.dart';
+import 'package:illinois/ui/academics/OccupationDetails.dart';
+import 'package:illinois/ui/widgets/HeaderBar.dart';
+import 'package:illinois/utils/Utils.dart';
+import 'package:rokwire_plugin/service/localization.dart';
+import 'package:rokwire_plugin/service/styles.dart';
+import 'package:rokwire_plugin/ui/widgets/rounded_button.dart';
+
+class OccupationList extends StatelessWidget {
+  OccupationList({Key? key}) : super(key: key);
+
+  final List<Occupation> occupations = [
```

Comment:
I don't think these occupations are being used since you are loading occupations for your list from O*NET below.